packages/launchpad/src/action-components/share-site-modal/index.tsx

```diff
@@ -17,14 +17,15 @@ interface ShareSiteModalProps {
 const ShareSiteModal = ( { setModalIsOpen, site }: ShareSiteModalProps ) => {
 	const translate = useTranslate();
 	const queryClient = useQueryClient();
+	const siteSlug = site?.slug || ( site?.URL && new URL( site.URL ).host );
```

Comment:
The use of `||` here means that it's not obvious what values `siteSlug` might end up having, especially when `site.slug` and `site.URL` are not truthy. Might it be cleaner to make `siteSlug` null when we can't work out what the site slug is?
```suggestion
	const getSiteSlug = ( site ): string | null => {
		if ( ! site ) {
			return null;
		}
		
		if ( site.slug ) {
			return site.slug;
		}
		
		if ( site.URL ) {
			return ( new URL( site.URL ).host );
		}

		return null;
	}
	const siteSlug = getSiteSlug( site );
```